README.pl.md

```diff
@@ -0,0 +1,147 @@
+<p align="center"><a href="https://newpipe.net"><img src="assets/new_pipe_icon_5.png" width="150"></a></p> 
+<h2 align="center"><b>NewPipe</b></h2>
+<h4 align="center">Wolny, lekki streamingowy frontend na Androida.</h4>
+
+<p align="center"><a href="https://f-droid.org/packages/org.schabi.newpipe/"><img src="https://fdroid.gitlab.io/artwork/badge/get-it-on-en.svg" alt="Pobierz na F-Droid" height=80/></a></p>

```

Comment:
We want a localized badge here. Also, I changed the alt text to be the same as the badge's text to be more consistent, but if that's wrong somehow then you can change it back.

```suggestion
<p align="center"><a href="https://f-droid.org/packages/org.schabi.newpipe/"><img src="https://fdroid.gitlab.io/artwork/badge/get-it-on-pl.svg" alt="Pobierz z F-Droid" height=80/></a></p>
```